apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.test.js

```diff
@@ -223,19 +224,6 @@ describe('RegisterMultiPaymentService', () => {
                     await registerMultiPaymentByTestClient(commonData.client, payload)
                 }, REGISTER_MP_NO_ACQUIRING_CONSUMERS)
             })
-            test('All ServiceConsumers should have BillingAccount', async () => {
```

Comment:
it looks like you need to add a test for Receipts. The Resident should see mote receipts for one Sevice Consumer.

Reply:
Ok will write todo